ui/components/src/components-react/properties/renderers/PropertyRenderer.tsx

```diff
@@ -48,7 +48,7 @@ export interface SharedRendererProps {
   /** Width of the whole property element */
   width?: number;
   /** Array of action button renderers
-   * @beta */
+   * @public */
```

Comment:
This one tag should be removed too. 

I have looked in ui-components package and did not find anymore unnecessary tags on properties. I haven't looked in other ui packages